app/controllers/investigations_controller.rb

```diff
@@ -63,7 +63,12 @@ def create
     if @investigation.save
       respond_to do |format|
         flash[:notice] = "The #{t('investigation')} was successfully created."
-        format.html { redirect_to investigation_path(@investigation) }
+        if params[:redirect]
```

Comment:
Yes, because I'm using Create Investigation in File Manager also. (I wanted to keep previous Create Investigation page, so here I added this line to prevent redirecting to the investigation page )